deploy/charts/rook-ceph/templates/servicemonitor.yaml

```diff
@@ -0,0 +1,22 @@
+{{- if and .Values.serviceMonitor.enabled (or .Values.csi.enableGrpcMetrics .Values.csi.enableLiveness) }}
```

Comment:
We also need to check if `monitoring.enabled` is true, otherwise we can't assume prometheus types like the ServiceMonitor are available.